os/include/tinyara/common_logs/common_logs.h

```diff
@@ -0,0 +1,84 @@
+typedef enum
+{
+	/* Memory Operations */
+	CMN_LOG_ALLOC_FAIL,
+	CMN_LOG_NULL_CHECK_FAIL,
+	CMN_LOG_OUT_OF_MEMORY,
+	CMN_LOG_MEM_LEAK,
+	CMN_LOG_BUFFER_OVERFLOW,
+
+	/* File Operations */
+	CMN_LOG_FILE_NFOUND,
+	CMN_LOG_FILE_OPEN_ERROR,
+	CMN_LOG_FILE_READ_ERROR,
+	CMN_LOG_FILE_WRITE_ERROR,
+	CMN_LOG_FILE_SEEK_ERROR,
+	CMN_LOG_FILE_IOCTL_ERROR,
+
+	/* Network Operations */
+	CMN_LOG_CON_TIMEOUT,
+	CMN_LOG_CON_REFUSED,
+	CMN_LOG_DATA_TRANS_ERROR,
+	CMN_LOG_HANDSHAKE_FAILED,
+	CMN_LOG_CERTIFICATE_INVALID,
+
+	/* Authentication and Authorization */
+	CMN_LOG_INVALID_CREDENTIALS,
+
+	/* Commonly used logs for printing information */
+	CMN_LOG_VALUE_OF,
+	CMN_LOG_INVALID_VAL,
+	CMN_LOG_FAILED_OP,
+	CMN_LOG_PERFORMING_OP,
+	CMN_LOG_INIT_DONE,
+	CMN_LOG_DEINIT_DONE,
+	CMN_LOG_START,
+	CMN_LOG_END,
+	CMN_LOG_INVALID_DATA,
+	CMN_LOG_VALID_DATA,
+	CMN_LOG_NOT_SUPPORTED,
+
+	CMN_LOG_MSG_MAX	
+
+} CLOG_INDEX;
+
+static const char* clog_message_str[CMN_LOG_MSG_MAX] =
+{
+	/* Memory Operations */
+        "Memory allocation failed :",	//CMN_LOG_ALLOC_FAIL,
+        "Null pointer check failed :",	//CMN_LOG_NULL_CHECK_FAIL,
+        "Out of memory :",		//CMN_LOG_OUT_OF_MEMORY,
```

Comment:
What is difference between 1st and this?
memory allocation failed and out of memory

Reply:
I removed the out of memory